source/reference/access-mdb-shell-help.txt

```diff
@@ -31,7 +31,10 @@ To see the list of options and help for starting ``mongosh``, use the
 Shell Help
 ----------
 
-To see the list of help, in ``mongosh``, type ``help``:
+.. include:: /includes/admonitions/help-interchangeable.rst
```

Comment:
I don't love how this is nested under `Shell Help` despite applying to Databases and Collection help as well. As written it's not immediately clear that it applies to all sections on this page.

One way to address this may be to make `Database Help` and `Collection Help` sub-sections of `Shell Help`. Alternatively, we pull this blurb up to the introduction and clarify that it applies when accessing Database and Collection help methods.

Reply:
tried this a few different ways. Didn't like `Database` and `Collection` help as subheadings to `Shell`. Added to the intro, clarified language, and added links.